Commands/Public/contacts.js

```diff
@@ -23,15 +23,15 @@ module.exports = async(client, msg, suffix) => {
 	});
 
 	// REMOVE THIS WHEN FIXED
-	msg.channel.send({ embed: { color: config.colors.info, title: "Caution", description: `This command may cause issues in some circumstances. \nIf you are unable to use the bot hereafter, join our [support server](${config.guildInvite}) and tell us the steps you performed.` } });
+	msg.channel.send({ embed: { color: config.colors.error, title: "Caution", description: `This command may cause issues in some circumstances.\nIf you are unable to use the bot hereafter, [join our support server](${config.guildInvite}) and tell us the steps you performed.` } });
```

Comment:
It's an info embed, not an error embed.

Reply:
My logic was that "Caution" would fit better with red than blue but sure

Reply:
I understand, but yellow/orange would be better suited for that. Nothing went wrong, so error would be going too far.

Reply:
yellowpages maybe? or is that too light

Reply:
I wouldn't link it to a menu color. Info is info, error is error, yellow pages is yp... so then I guess orange is the best option. (Not pure orange please)